### ISSUE TITLE
Descriptions in Tool

### ISSUE THREAD

**Issue by Keristiena** [uno-isqa-8950/uno-cpi#1466]
The tool is not necessarily telling people what it is doing, when it is doing it. So if someone searches and there are no results, it doesn't say no results (it just doesn't do anything). 

We need a description/confirmation for user interactions

@robynlloos

**Comment by robynlloos** [comment]
Agreed - I asked LaTrina for an example of what she noticed to help clarify (if she can recall what she was searching at that time). I think we just nee some sort of "No Results" pop up or text that goes into the area where people are searching. 

Could the students provide some suggestions to this that are typically used in sites?
